src/App.jsx

```diff
@@ -1,10 +1,71 @@
 import './App.css';
+import axios from "axios";
+import {useState} from "react";
+import areaColor from "./helpers/areaColor.js";
+import CountryInformation from"./assets/country-information/country-information.jsx";
+
+// onClick verandert de classname van de button en de info over de landen
+// --> button gaat weg en landen worden getoond
+
+
 
 function App() {
+    const [data, setData] = useState(null)
+    const [sortedData, setSortedData] = useState(null)
```

Comment:
Voor het gemak zou ik hier gaan voor een default waarde als een lege array. Dan kun je je code verderop wat versimpelen en hoef je geen null checks te doen
```suggestion
    const [sortedData, setSortedData] = useState([]);
```